webapp/src/components/call_widget/component.tsx

```diff
@@ -905,13 +907,13 @@ export default class CallWidget extends React.PureComponent<Props, State> {
 
         if (el) {
             this.audioMenuResizeObserver = new ResizeObserver((entries) => {
-                if (entries.length === 0) {
+                if (entries.length === 0 || entries[0].borderBoxSize.length === 0) {
                     return;
                 }
                 sendDesktopEvent('calls-widget-resize', {
                     element: 'calls-widget-audio-menu',
-                    width: Math.round(entries[0].contentRect.width),
-                    height: Math.round(entries[0].contentRect.height),
+                    height: Math.round(entries[0].borderBoxSize[0].blockSize),
+                    width: Math.round(entries[0].borderBoxSize[0].inlineSize),
```

Comment:
Changes here are needed to make sure the width passed to the desktop side accounts for the scrollbar if present.